catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/FormulaTest.java

```diff
@@ -208,7 +216,8 @@ public void stringConcatenationTest() {
 		FormulaElement joinFunctionFormulaElement = new FormulaElement(ElementType.FUNCTION,
 				Functions.JOIN.name(), null, helloStringFormulaElement, worldStringFormulaElement);
 		Formula joinFormula = new Formula(joinFunctionFormulaElement);
-		String computeDialogResult = joinFormula.getResultForComputeDialog(null);
+		StringProvider stringProvider = mock(StringProvider.class);
```

Comment:
Please try to put duplicate code into setup if possible

Reply:
Unfortunately, the StringProvider is not used in all of the tests (approximately 50%)